src/Abp.Zero.EntityFramework/EntityHistory/EntityHistoryHelper.cs

```diff
@@ -487,6 +509,33 @@ private void UpdateChangeSet(DbContext context, EntityChangeSet changeSet)
                 var entityType = GetEntityType(context.As<IObjectContextAdapter>().ObjectContext, entityEntry.GetEntityBaseType(), useClrType: false);
                 entityChange.EntityId = GetEntityId(entityEntry, entityType);
 
+                /* Update audit field */
+                var removePropertyChanges = new List<EntityPropertyChange>();
+                foreach (var propertyChange in entityChange.PropertyChanges.
+                    Where(x => IsAuditField(x.PropertyName)))
+                {
+                    var propertyEntry = entityEntry.Property(propertyChange.PropertyName);
+                    if (propertyEntry != null)
+                    {
+                        if (propertyChange.NewValue == propertyEntry.GetNewValue().ToJsonString().TruncateWithPostfix(EntityPropertyChange.MaxValueLength) &&
+                            propertyChange.OriginalValue == propertyChange.NewValue)
+                        {
+                            removePropertyChanges.Add(propertyChange);
+                        }
+                        else
+                        {
+                            propertyChange.NewValue = propertyEntry.GetNewValue().ToJsonString()
```

Comment:
can help to add `TruncateWithPostfix` here as well.

https://github.com/aspnetboilerplate/aspnetboilerplate/blob/df144d13800537d0e21734098306ee47dac848ec/src/Abp.Zero.EntityFramework/EntityHistory/EntityHistoryHelper.cs#L509-L515